sdks/python/build.gradle

```diff
@@ -84,6 +84,18 @@ task buildPython(dependsOn: 'setupVirtualenv') {
 }
 build.dependsOn buildPython
 
+// Snapshot of dependency requirements defined in setup.py.
+// Results will be stored in files under Gradle build directory.
+task depSnapshot(dependsOn: 'installGcpTest') {
+  doLast {
+    println 'Snapshoting full dependencies requirements with versions info to requirements.txt.'
+    exec {
+      executable 'sh'
+      args '-c', ". ${envdir}/bin/activate && pip freeze --local > ${project.buildDir}/requirements.txt"
```

Comment:
You can also add `--all` to unhide pip/wheel/setuptools versions.

Reply:
sg.